### PR TITLE
fix(bundle): remove inline @github deps

### PR DESCRIPTION
**Pull request by joshblack**
This PR updates the rollup config to not inline `@github` dependencies in CommonJS. This is due to two problems when packaging:

- The `node_modules` folder when `lib` is ignored when publishing
- The `require()` statement within `_FormattingTools.tsx` is not being rewritten to the inlined dependency when enabled

To unblock 35.11, this PR reverts this change back to the existing behavior until we can resolve the two points above.

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -69,11 +69,7 @@ export default [
   // CommonJS
   {
     ...baseConfig,
-    external: isExternal(
-      external.filter(id => {
-        return !id.startsWith('@github')
-      })
-    ),
```
[the remark this answers is not in the document]
Comment (joshblack) [reply rollup.config.js:76]: Hey @pksjce! 👋 

The background for the filter was to inline the ESM-only dependencies from `@github` in the CommonJS bundle to get that working again. This was kind of a trade-off until we moved to ESM-only and don't have to support CommonJS anymore

**Comment by joshblack** [comment]
Following up on this, it seems like it may be due to the workflow publishing using `yarn` which is not including files in `files` 🤔 

The canary publishes that use `changeset` work as-intended so the update here may need to be to update the RC workflow to mirror the canary one

**Comment by broccolinisoup** [comment]
This looks good to me!

File: rollup.config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (pksjce) [reply rollup.config.js:76]: As long as we are fine with ESM only, LGTM!

**Pull request merged**